kiwi/system/result.py

```diff
@@ -15,45 +15,53 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-from collections import namedtuple
 import logging
 import pickle
 import os
+from typing import (
+    Dict, NamedTuple, TypeVar
+)
 
 # project
+from kiwi.xml_state import XMLState
 from kiwi.exceptions import (
     KiwiResultError
 )
 
 log = logging.getLogger('kiwi')
 
 # must be global to allow pickle to find it
-result_file_type = namedtuple(
-    'result_file_type', ['filename', 'use_for_bundle', 'compress', 'shasum']
+ResultFileType = NamedTuple(
```

Comment:
Please keep the naming as ```result_file_type``` and the indentation as suggested in former comments

Reply:
I will change it.

The reason for not choosing snake case naming was that I get an info message from mypy
```
 Class name "result_file_type" doesn't conform to PascalCase naming style
 ```